test/integration/graql/query/GraqlDeleteIT.java

```diff
@@ -110,527 +110,8 @@ public static void closeSession() {
-
-        Role author = tx.putRole("author");
-        Role work = tx.putRole("work");
-        RelationType authoredBy = tx.putRelationType("authored-by").relates(author).relates(work);
-        EntityType person = tx.putEntityType("person").plays(author);
-        EntityType production = tx.putEntityType("production").plays(work);
-        AttributeType<String> provenance = tx.putAttributeType("provenance", AttributeType.ValueType.STRING);
-        authoredBy.has(provenance);
-
-        Entity aPerson = person.create();
-        Relation aRelation = authoredBy.create();
-        aRelation.assign(author, aPerson);
-        Attribute<String> aProvenance = provenance.create("hello");
-        aRelation.has(aProvenance);
-        ConceptId relationId = aRelation.id();
-
-        tx.commit();
-        tx = session.transaction(Transaction.Type.WRITE);
-
-        aProvenance = tx.getAttributesByValue("hello").iterator().next();
-        assertTrue(aProvenance.type().label().toString().equals("provenance"));
-        assertEquals(aProvenance.owners().count(), 1);
-
-        aRelation = tx.getConcept(relationId);
-        aRelation.delete();
-        assertTrue(aRelation.isDeleted());
-
-        assertEquals(aProvenance.owners().collect(Collectors.toList()).size(), 0);
-    }
-
-    @Test
-    public void deleteLastRolePlayer_RelationAndAttrOwnershipIsRemoved() {
-        Session session = graknServer.sessionWithNewKeyspace();
-        Transaction tx = session.transaction(Transaction.Type.WRITE);
-
-        Role author = tx.putRole("author");
-        Role work = tx.putRole("work");
-        RelationType authoredBy = tx.putRelationType("authored-by").relates(author).relates(work);
-        EntityType person = tx.putEntityType("person").plays(author);
-        EntityType production = tx.putEntityType("production").plays(work);
-        AttributeType<String> provenance = tx.putAttributeType("provenance", AttributeType.ValueType.STRING);
-        authoredBy.has(provenance);
-
-        Entity aPerson = person.create();
-        ConceptId personId = aPerson.id();
-        Relation aRelation = authoredBy.create();
-        aRelation.assign(author, aPerson);
-        Attribute<String> aProvenance = provenance.create("hello");
-        aRelation.has(aProvenance);
-        ConceptId relationId = aRelation.id();
-
-        tx.commit();
-        tx = session.transaction(Transaction.Type.WRITE);
-
-        aProvenance = tx.getAttributesByValue("hello").iterator().next();
-        assertTrue(aProvenance.type().label().toString().equals("provenance"));
-        assertEquals(aProvenance.owners().count(), 1);
-
-        aRelation = tx.getConcept(relationId);
-        tx.getConcept(personId).delete();
-        assertTrue(aRelation.isDeleted());
-
-        assertEquals(aProvenance.owners().collect(Collectors.toList()).size(), 0);
-    }
-
-
-    @Test
-    public void afterDeletingAllInstances_TheTypeCanBeUndefined() {
```

Comment:
have we got this in `Undefine` already? Otherwise, we shouldn't delete it yet

Reply:
Thought I added it but apparently not! Will merge https://github.com/graknlabs/verification/pull/52 and then add it to undefine.feature.

Reply:
... OK, never mind, it was in master but not in my local repo somehow.

`undefine.feature` has a draft scenario named:
`Scenario: a type can be undefined after deleting all of its instances`